### ISSUE TITLE
upgrade new relic dependencies to the same version

### ISSUE THREAD

**Issue by bwiercinski** [kamon-io/Kamon#995]
currently `telemetry` package is in version `0.10.0` and `telemetry-http-okhttp` is in version `0.12.0`

https://github.com/SimunKaracic/Kamon/blob/master/build.sbt#L587-L588

because new relic renamed `telemetry` package to `telemetry-core`. now i have

```
com/newrelic/telemetry/telemetry-core/0.12.0/telemetry-core-0.12.0.jar
com/newrelic/telemetry/telemetry/0.10.0/telemetry-0.10.0.jar
```

in my classpath. this causing that i'm unable to assembly my service into JAR without workarounds.

could you upgrade `telemetry:0.10.0` to `telemetry-core:0.12.0` and release new version of kamon?

**Comment by bwiercinski** [comment]
https://github.com/kamon-io/Kamon/pull/966

**Comment by SimunKaracic** [comment]
Thanks for the tip, I had no idea they renamed the package...

**Comment by bwiercinski** [comment]
it is okey, many thanks for fast reply :)

**Comment by SimunKaracic** [comment]
Fixed in 2.1.16!
